package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "xcomponentjsapi",
+  "version": "1.0.0",
+  "description": "js api",
```

Comment:
Give a more accurate description
ex: Javascript reactive client API for XComponent